apis/projectcontour/v1/httpproxy.go

```diff
@@ -18,6 +18,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// HttpVersion is an alias to enforce validation
+// +kubebuilder:validation:Enum=h2;http/1.1
+type HttpVersion string
```

Comment:
Please use the following casing for the Go type/field:
```suggestion
type HTTPVersion string
```